dspace-server-webapp/src/main/java/org/dspace/app/rest/security/SubmitterReadPermissionEvaluatorPlugin.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.security;
+
+import java.io.Serializable;
+import java.sql.SQLException;
+import java.util.UUID;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.app.rest.model.ItemRest;
+import org.dspace.app.rest.utils.ContextUtil;
+import org.dspace.content.Item;
+import org.dspace.content.service.BitstreamService;
+import org.dspace.content.service.BundleService;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.dspace.eperson.EPerson;
+import org.dspace.services.RequestService;
+import org.dspace.services.model.Request;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+
+/**
+ * {@link RestObjectPermissionEvaluatorPlugin} class that evaluates if eperson is submitter of an item and authorize
+ * accordingly.
+ *
+ * @author Bui Thai Hai (thaihai.bui@dlcorp.com.vn)
+ */
+
+
+//TODO: rename this to fit the use case
+@Component
+public class SubmitterReadPermissionEvaluatorPlugin extends RestObjectPermissionEvaluatorPlugin {
```

Comment:
I'm fine with that as it is based on my suggestion https://github.com/DSpace/DSpace/issues/8182#issuecomment-1285350688 and what we have implemented in DSpace-CRIS  but I guess that it would deserve a wide discussion as it would break the rules followed by DSpace so far, i.e. base security decision exclusively on what is written in the resource policies.
If we agree to move in this direction we will be happy to donate what has been implemented in DSpace-CRIS for DSpace 8. In such case I suggest to limit this PR to the resolution of the issue 8272

Reply:
Then I will revert the commit related to #8182, no need to work on it any further. I hope to see your solution since I failed to find it in the CRIS repo.